### ISSUE TITLE
Figure out a way to use bundler like webpack in the repo

### ISSUE THREAD

**Issue by eswarsaladi** [NEU-Libraries/drs-toolkit-wp-plugin#273]
## Why is it being proposed?
1. Imports and exports can be used in the code
2. The code can be compiled into one js file reducing the overall build size and speed of the plugin

## Important Links to check out
1. https://parceljs.org/
2. https://webpack.js.org/
3. https://babeljs.io/

**Comment by patrickmj** [comment]
yes! research this, and we will talk about what you discover when we meet next. all kinds of performance issues have plagued CERES for years. if we can show work on that, it'll be fantastic. probably a good thing to talk about in the demo to dev meeting.
